app/assets/v2/js/pages/bounty_details.js

```diff
@@ -373,6 +373,16 @@ pull_interest_list(result['pk'], function(is_interested){
         }
         actions.push(entry);
 
+        var enabled = isBountyOwner(result);
+        var addContributionEntry = {
+            href: '/funding/increase?source='+result['github_url'],
+            text: 'Add Contribution',
+            parent: 'right_actions',
+            color: enabled ? 'darkBlue' : 'darkGrey',
+            extraClass: enabled ? '' : 'disabled',
+            title: enabled ? 'Increase the funding of this bounty' : 'Can only be performed if you are the funder.'
```

Comment:
cant wait to get a v2 out there so that we can remove `Can only be performed if you are the funder`!